src/tests/__mocks__/timebankMocks.ts

```diff
@@ -80,7 +80,7 @@ export const personsMock2: Person[] = [
     startDate: "test",
     unspentVacations: 0,
     spentVacations: 0,
-    minimumBillableRate: 0
+    minimumBillableRate: 50
```

Comment:
Was there a reason for changing this value? I don't see it being asserted in any tests?